boxmot/trackers/hybridsort/hybridsort.py

```diff
@@ -408,10 +408,7 @@ def update(self, dets, im):
         ret = []
         for t, trk in enumerate(trks):
             pos, kalman_score, simple_score = self.trackers[t].predict()
-            try:
-                trk[:6] = [pos[0][0], pos[0][1], pos[0][2], pos[0][3], kalman_score, simple_score[0]]
```

Comment:
I am pretty sure this did not work (anymore).
Changing it as recommended here https://github.com/mikel-brostrom/yolo_tracking/issues/1221#issuecomment-1852896811, just results in the exception case every time.

Based on the return cases in predict, I am decently sure it always has to be 
`   trk[:6] = [pos[0][0], pos[0][1], pos[0][2], pos[0][3], kalman_score[0], simple_score]
`
without any fallback necessary.

But maybe I am missing something.

Reply:
> Based on the return cases in predict, I am decently sure it always has to be
>   trk[:6] = [pos[0][0], pos[0][1], pos[0][2], pos[0][3], kalman_score[0], simple_score]
> without any fallback necessary.

This was taken straight out of the original repo. Not sure if it is needed tbh.

Reply:
Let's keep it like this for now :smile:

Reply:
Yeah, I checked. But it's unclear why that should work :shrug: